Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "9fefc737155e789ced61b41750b4273c7780ac7801c50cf36dc5925be3b85783"
+            "sha256": "0242e3e296e09b30fb69e0d7a2f2e8feb4c6a23d3c7ec99500f2883a032a8c84"
```

Comment:
Should probably try not to update everything in this file.  I think it will need manual help to revert unrelated things

Reply:
It's very weird how pipenv just defaults to updating _everything_... I tried to fix this with `--keep-outdated`, let me know if it's better